src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/Delegation/DelegatedCompletionItemResolver.cs

```diff
@@ -96,13 +96,13 @@ internal class DelegatedCompletionItemResolver : CompletionItemResolver
             {
                 return resolvedCompletionItem;
             }
-            
-            // TODO: Pull active formatting options from client.
```

Comment:
To remove this TODO is the primary goal of this PR. Aka, ask client for formatting options so when we format snippets the right thing happens.